src/test/java/space/obminyashka/items_exchange/controller/UserControllerTest.java

```diff
@@ -298,4 +298,15 @@ private User createUser() {
 
         return user;
     }
+
+    @Test
+    @WithMockUser(username = "admin")
+    void addChild_InvalidChildAge_ShouldReturnHttpStatusBadRequest() throws Exception {
+        var childDto = getTestChildren(0L, 0L, 2001);
+        when(userService.findByUsernameOrEmail(any())).thenReturn(Optional.of(user));
+
+        final MvcResult mvcResult = sendDtoAndGetMvcResult(post(USER_CHILD), childDto, status().isBadRequest());
+        assertTrue(Objects.requireNonNull(mvcResult.getResolvedException()).getMessage()
```

Comment:
It would be mush better to make it with 'assertAll() wrapper 'assertNotNull' and 'assertTrue' inside. Additionally, you can check the exception class if that is provided.